src/renderer/modals/Send/index.js

```diff
@@ -5,10 +5,13 @@ import Modal from "~/renderer/components/Modal";
 import Body from "./Body";
 import type { StepId } from "./types";
 
-class SendModal extends PureComponent<{}, { stepId: StepId }> {
-  state = {
-    stepId: "recipient",
-  };
+class SendModal extends PureComponent<{ stepId: StepId, onClose: Function }, { stepId: StepId }> {
```

Comment:
I would appreciate if you convert it to pure component if the component logic is this simple.

Reply:
Could you avoid using general Function type? It'll become a pain when migrating to TS. `$PropertyType<Modal, 'props'>['onClose']` would do the job i believe.

Reply:
ok will change

Reply:
i did not change because i dont like the naming of the props stepId that interfere with the state